Directory.Build.props

```diff
@@ -0,0 +1,46 @@
+<Project>
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Product>MQTTnet</Product>
+        <Authors>The contributors of MQTTnet</Authors>
+        <Company>The contributors of MQTTnet</Company>
+        <Copyright>Copyright (c) .NET Foundation and Contributors</Copyright>
+
+        <DelaySign>false</DelaySign>
+        <SignAssembly>false</SignAssembly>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <EnableNETAnalyzers>false</EnableNETAnalyzers>
+        <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
+
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <NoWarn>1591;NETSDK1138;NU1803;NU1901;NU1902</NoWarn>
+
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <NuGetAuditMode>all</NuGetAuditMode>
+        <NuGetAudit>true</NuGetAudit>
+        <NuGetAuditLevel>low</NuGetAuditLevel>
+        <AnalysisLevel>latest-Recommended</AnalysisLevel>
+
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/dotnet/MQTTnet</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/dotnet/MQTTnet.git</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
After enabling ContinuousIntegrationBuild, it affects the lower breakpoint of the upper-level project in Debug configuration. Can we add a condition here to enable it only in Release configuration?